wpimath/src/main/java/edu/wpi/first/math/controller/PIDController.java

```diff
@@ -228,17 +228,37 @@ public double getPeriod() {
    *
    * @return the position tolerance of the controller.
    */
+  @Deprecated(forRemoval = true, since = "2025")
   public double getPositionTolerance() {
-    return m_positionTolerance;
+    return m_error;
   }
 
   /**
    * Returns the velocity tolerance of this controller.
    *
    * @return the velocity tolerance of the controller.
```

Comment:
Add `@deprecated` attributes like `@deprecated Use getErrorDerivativeTolerance() instead.` to each deprecated function so the user knows what to replace it with.